### DISCUSSION TITLE
zzh! CC2652R issues

### DISCUSSION THREAD

**Discussion by Laphroa1g** [Koenkk/zigbee2mqtt#5941] [Devices]
Hi

I’m having a lot of problems with a zzh! CC2652R, and hoping someone can help.
I have 64 devices and they all work OK, but after a random amount of time which is usually less than 24 hours, zigbee2mqtt cannot communicate with the co-ordinator and it all stops.

Usually just removing it then plugging back in gets it working again, or sometimes have to also restart zigbee2mqtt. It was initially used on a Pi 3 and tried in different physical USB ports, but it is now connected to an Unraid server hoping that the Pi was the issue, but the same is still happening. The Unraid servver has had a Z Wave stick plugged in for the last 18 months with no issues, although there are only 3 devices on that network. 
Yesterday I wiped the flash and re-installed the firmware but have now run out of ideas.
What should I be looking for in the logs? 
Or is it just faulty? I'd order another one but just to get a quick fix, but looking on Tindie there is no stock currently. 

Any advise is much appreciated.

**Comment by Koenkk** [comment]
- Is the zzh connected through a usb extension cable? (if not do so).
- Is modemmanager installed? https://www.zigbee2mqtt.io/information/FAQ.html#modemmanager-is-installed
- Do you run openhab with the zwave binding? https://www.zigbee2mqtt.io/information/FAQ.html#for-openhab-users-disable-zwave-binding

**Reply by Laphroa1g** [reply]
Thanks for the reply.

I had an extension lead connected for the last 2 days but only since moving it to the Unraid server.
Modem manager is not installed on the Pi, but I don't think this will be running on Unraid.
I am using Homeassistant, which is running in a docker on Unraid.
To simplify things, I've moved the stick back to the Pi using the USB extension lead and will see how long it lasts and report back.

**Reply by Laphroa1g** [reply]
It worked OK for about 30 hours, but now some of the devices are still working but most have stopped. I didn't re-pair everything 2 days ago and just concentrated on the motion sensors, light switches, light bulbs and the automated blind. The Aqara motion sensors are still reporting motion but none of the lights, switches or the blind are working.  

I use Zigbee2MqttAssistant and in there it is still reporting that the bridge state is online and:

"Version of Zigbee2MqttAssistant (this software): 0.3.157 (0.3.157+Branch.master.Sha.ea388c3bbc2d43c4c9c62f27bb6764911ba4acc9) Release
Version of Zigbee2Mqtt: 1.17.0
Coordinator version: 20201026
Coordinator Zigbee Id: 0x00124b0021b77b54"

Usually everything stops working and the Coordinator version and Coordinator Zigbee Id are blank. Another difference this time is that the "Allow new devices to join network" button isn't working either. I tried to re-pair a light switch and a bulb but that didn't work.

I'm itching to just pull the stick out and in again to get it all working, but I've left it in this state at the moment and can download any logs that would help identify the issue.

Thanks for you help.

**Reply by Laphroa1g** [reply]
Now the motion sensors have stopped working and the coordinator is not seen.

 Version of Zigbee2MqttAssistant (this software): 0.3.157 (0.3.157+Branch.master.Sha.ea388c3bbc2d43c4c9c62f27bb6764911ba4acc9) Release
Version of Zigbee2Mqtt:
Coordinator version:
Coordinator Zigbee Id: 

A snippet of Logs, I can download whatever you suggest

Jan 29 13:00:15 officepi npm[19004]: Zigbee2MQTT:info  2021-01-29 13:00:15: MQTT publish: topic 'zigbee2mqtt/bridge/log', payload '{"message":"Publish 'set' 'brightness' to '0x588e81fffe2c0279' failed: 'Error: Command 0x588e81fffe2c0279/1 genLevelCtrl.moveToLevelWithOnOff({\"level\":253,\"transtime\":0}, {\"timeout\":10000,\"disableResponse\":false,\"disableRecovery\":false,\"disableDefaultResponse\":false,\"direction\":0,\"srcEndpoint\":null,\"reservedBits\":0,\"manufacturerCode\":null,\"transactionSequenceNumber\":null}) failed (SRSP - AF - dataRequest after 6000ms)'","meta":{"friendly_name":"0x588e81fffe2c0279"},"type":"zigbee_publish_error"}'
Jan 29 13:01:03 officepi npm[19004]: Zigbee2MQTT:error 2021-01-29 13:01:03: Publish 'set' 'state' to '0x00178801040222a9' failed: 'Error: Command 0x00178801040222a9/11 genOnOff.off({}, {"timeout":10000,"disableResponse":false,"disableRecovery":false,"disableDefaultResponse":false,"direction":0,"srcEndpoint":null,"reservedBits":0,"manufacturerCode":null,"transactionSequenceNumber":null}) failed (Data request failed with error: 'Timeout' (9999))'
Jan 29 13:01:03 officepi npm[19004]: Zigbee2MQTT:info  2021-01-29 13:01:03: MQTT publish: topic 'zigbee2mqtt/bridge/log', payload '{"message":"Publish 'set' 'state' to '0x00178801040222a9' failed: 'Error: Command 0x00178801040222a9/11 genOnOff.off({}, {\"timeout\":10000,\"disableResponse\":false,\"disableRecovery\":false,\"disableDefaultResponse\":false,\"direction\":0,\"srcEndpoint\":null,\"reservedBits\":0,\"manufacturerCode\":null,\"transactionSequenceNumber\":null}) failed (Data request failed with error: 'Timeout' (9999))'","meta":{"friendly_name":"0x00178801040222a9"},"type":"zigbee_publish_error"}

**Reply by walthowd** [reply]
It seems to be an issue with Z-Stack locking up -- I've been chasing it for a few months now. I can normally recreate it on my 120 node network in a period of 1 to 8 days. Same lock up happens with CC1352P2 using the onboard XDS debugger and with the ZZH using the CH340 driver. 

When the lockup occurs, you don't anything back from Z-Stack it's just completely dead. Serial port still looks good, nothing from dmesg, etc. 

Fairly sure root cause is something in the Z-Stack firmware that TI will have to fix and then @Koenkk will have to include in future releases.

Seems to be more reports from x86 users then anyone else -- Also seems more prevalent with larger networks.
